frontend/src/pages/home-page.tsx

```diff
@@ -30,15 +32,28 @@ function HomePage() {
         <div className="absolute inset-0 bg-black opacity-50"></div>
         <div className="absolute inset-0 flex flex-col px-8 py-8 text-slate-50 md:px-16">
           <div className="flex w-full justify-between">
-            <div className="text-2xl font-semibold">WanderLust</div>
+            <div className="text-2xl font-semibold flex justify-between items-center">WanderLust</div>
+            <div className='flex justify-between px-2'>
+            <div className="flex justify-end py-2 px-2 md:px-20">
+              <ThemeToggle />
+            </div>
             <button
-              className="rounded border border-slate-50 px-4 py-2 hover:bg-slate-500/25"
+              className="hidden md:inline-block rounded border border-slate-50 px-4 py-2 hover:bg-slate-500/25"
               onClick={() => {
                 navigate('/add-blog');
               }}
             >
               Create post
             </button>
+            <button
+              className="md:hidden px-2 py-2 hover:bg-slate-500/25"
+              onClick={() => {
+                navigate('/add-blog');
+              }}
+            >
+             <svg className="h-8 w-8 text-white"  viewBox="0 0 24 24"  fill="none"  stroke="currentColor"  strokeWidth="2"  strokeLinecap="round"  strokeLinejoin="round">  <rect x="3" y="3" width="18" height="18" rx="2" ry="2" />  <line x1="12" y1="8" x2="12" y2="16" />  <line x1="8" y1="12" x2="16" y2="12" /></svg>
```

Comment:
Kindly take this out as a svg component and place it where all the svgs are placed and then pull it as a component